src/metadatas.rs

```diff
@@ -163,3 +180,19 @@ fn test_stop_need_phone_driver() {
     assert!(!metadatas.some_stops_need_phone_agency);
     assert!(metadatas.some_stops_need_phone_driver);
 }
+
+#[test]
+fn test_route_have_colors() {
+    let raw_gtfs =
+        gtfs_structures::RawGtfs::new("test_data/no_fares_no_shapes").expect("Failed to load data");
```

Comment:
Wondering why you expect to have the message "Failed to load data"

Reply:
I agree the `expect` syntax is misleading. It is the message you will get if you fail to load the GTFS data.
https://stackoverflow.com/questions/66362625/why-is-rusts-expect-called-expect

Reply:
Thanks!